### PR TITLE
Created Uganda Annual Culture Forum page  https://www.loom.com/share/40a6700159b74234983f5c9c6cce1206

### PR DESCRIPTION
**Pull request by danielmatama**
I created the mobile version of the Uganda Annual Culture homepage using HTML and styled it using CSS
I made the mobile version of the about page
I made the desktop version of the home page and the desktop version of the about page
I made links to connect the pages and a dropdown button for mobile versions.
I input the speakers' section dynamically in javascript

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @danielmatama 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: 

You have submitted a project, that doesn't run linter on the PR. Kindly make sure the linters are running on your PR to check your code.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,244 @@
...
+
+> Add a link to your deployed project.
+
+- [https://danielmatama.github.io/matama-capstone/](https://danielmatama.github.io/matama-capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
```
Comment (fobadara) [inline README.md]: [REQUIRED] I can't seem to find the link to your video presentation. I had to type in the link to view the page. Kindly include it

File: README.md [hunk 2]
```diff
@@ -1 +1,244 @@
...
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (fobadara) [inline README.md]: [REQUIRED] Could you credit Cindy shin for the design? Don't forget to include a link to her work

File: index.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+      <li class="nav-link mobile-link">
+        <a href="#" class="dropdown-button">
+          <img src="images/menubar.svg" width="24px" height="24px" alt="decorative image">
+        </a>
+      </li>
+      </div>
+
+      <div class="dropdown-menu hide" id="myDropdown">
+        <span class="xmark"><i class="fa-solid fa-xmark"></i></span>
+        <a href="about-page/about.html" class="ax">About</a>
```
Comment (fobadara) [inline index.html]: [REQIRED] Kindly ensure that the link to the about page is working. It currently redirects to a 404 page

File: about-page/about.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+</div>
+
+</section>
+
+ <!-- ....//copyright.../ -->
+<div class="copy">
+  <p>@danielmatama copyright</p>
+</div>
+
+<script src="scripts.js"></script>
```
Comment (fobadara) [inline about-page/about.html]: [REQUIRED] Kindly ensure that your footers contain a logo on the left and text on the right. One of the aims of this project is to teach you to follow project requirements.

[REQUIRED] I noticed that you are not using semantic tags like section, article, and footer among others in a lot of places. Kindly fix this.
[REQUIRED]For desktop screens the about page should have a dark background while the homepage should have white bacckground

File: about-page/about.html [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+      rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+
+    <!-- ......socials....// -->
+    <div class="socials">
+      <ul class="socials1">
+        <li class="social-img icon">
+          <img src="images/facebook.png" alt="twitter-icon" class="twitter">
```
Comment (fobadara) [inline about-page/about.html]: [REQUIRED] The content for your home and about page is overlaying.

new

File: scripts.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  <p class="f-para">Head Linguist</p>
+  <p class="f-para2">He has translated over 50 Ugandan languages to English</p>
+</div>
+</div>
+</div>
+
+<div class="f-hide">
+<div class="speaker">
+<div class="f-img">
+  <img src="images/checked.png" alt="" class="checked">
```
Comment (fobadara) [inline scripts.js]: [REQUIRED] Kindly arrange the needed data in an array of objects and create the elements from it. Something like this:

```
const speakerList = [ { name: "Speaker Name",... } ]

for (let i= ......) {
  const element = <p>speaker[I].name</p>
...........................
}
```

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @danielmatama, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+    <img src="images/red.png" alt="" class="art">
+    <!-- <a class="navbar-brand" href="#">navbar</a> -->
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <a class="nav-link active home" aria-current="page" href="/home/matama/matama-capstone/about-page/about.html">About</a>
+        </li>
```
Comment (binhussen) [inline index.html]: - Kindly update your navigation link as your previous code reviewer suggested, 
  - when you start from your current directory start with`./` and then to the folder or file
  - if you went to back start `../` 👌👌 
about directory `./about-page/about.html` 👌👍

File: about-page/about.html [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+    <a href="/home/matama/matama-capstone/index.html" class="arta"><img src="images/red.png" alt="" class="art"></a>
+    <!-- <img src="images/mother.png" alt="" class="art"> -->
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <!-- <div class="collapse navbar-collapse" id="navbarSupportedContent"> -->
+      <div class="lists">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <a class="nav-link active home" aria-current="page" href="/home/matama/matama-capstone/index.html">Home</a>
```
Comment (binhussen) [inline about-page/about.html]: - Kindly update your navigation link as your previous code reviewer suggested,
  - when you start from your current directory start with./ and then to the folder or file
  - if you went to back start ../ 👌👌
home directory`../index.html` 👌👍

File: README.md [hunk 9]
```diff
@@ -1 +1,252 @@
...
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Cindy Shin
+
```
Comment (binhussen) [inline README.md]: - [Optional] Great Job for adding an author in "Acknowledgements" section, I think it is better to make it with appropriate links together like  "  to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)."😉👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @danielmatama , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Suggested changes
- Kindly be sure your link is not broken 👍👍


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+      </ul>
+
+    </div>
+
+    <!-- ....//nav-bar..../// -->
+
+    <nav class="navbar navbar-expand-lg bg-light">
+  <div class="container-fluid">
+    <img src="images/red.png" alt="" class="art">
+    <!-- <a class="navbar-brand" href="#">navbar</a> -->
```
Comment (KayLemba) [inline index.html:49]: [OPTIONAL] Please consider deleting unused code for it makes your code base look untidy

File: index.html [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+   </div>
+</footer>
+
+<script src="scripts.js"></script>
+ </body>
+</html>
+
+
+
+<!-- <script src="scripts.js"></script> -->
```
Comment (KayLemba) [inline index.html:201]: [OPTIONAL] Please consider deleting unused code for it makes you code base look untidy

File: about.html [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+    <!-- ....//nav-bar..../// -->
+
+    <nav class="navbar navbar-expand-lg bg-light">
+  <div class="container-fluid">
+    <a href="./index.html" class="arta"><img src="images/red.png" alt="" class="art"></a>
+    <!-- <img src="images/mother.png" alt="" class="art"> -->
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <!-- <div class="collapse navbar-collapse" id="navbarSupportedContent"> -->
```
Comment (KayLemba) [inline about.html:52]: [OPTIONAL] Please kindly delete unused code fot it makes your code base look untidy

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @danielmatama         , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R





 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**